vendor/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -87,6 +87,11 @@ service RuntimeService {
     // Once it returns error, new container log file MUST NOT be created.
     rpc ReopenContainerLog(ReopenContainerLogRequest) returns (ReopenContainerLogResponse) {}
 
+    // CheckpointContainer checkpoints a container
+    rpc CheckpointContainer(CheckpointContainerRequest) returns (CheckpointContainerResponse) {}
```

Comment:
where did these two API come from? I didn't find it from cri-api repo.

Reply:
@feiskyer I was not aware of the correct process, but I now created a KEP for this: https://github.com/kubernetes/enhancements/pull/1990

Reply:
thanks, it needs to be in the cri-api first